srcpkgs/gammaray/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'gammaray'
+pkgname=gammaray
```

Comment:
Package name should probably be GammaRay. I believe repo name takes precedence over tarball.

Reply:
Oh, didn't see that earlier. IMHO the package name should be `gammaray` since that's the name of both the program binary and the source tarball.

Reply:
I'm not sure either. The name in its desktop file is GammaRay

Reply:
Please use `gammaray`

Reply:
My preference as a user would be "least surprise". The executable is called `gammaray`, so the package should be called that too. The user doesn't know about repos, and the application name in the desktop file is a (possibly localized) title for presentation purposes only.